### ISSUE TITLE
Scapulothoracic dissociation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11076]
**Preferred term label:**
Scapulothoracic dissociation

**Synonyms**

**Definition (free text, please give PubMed ID)**
Chest X-ray or CT showing >1cm difference in measurement of lateral scapular displacement from uninjured to injured sides, measuring the distance from the medial border of the scapula to the thoracic spinous process

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Scapulothoracic dissociation

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001272
- label: Scapulothoracic dissociation
- parents: [Abnormal scapula morphology (HP:0000782)](https://hpo.jax.org/app/browse/term/HP:0000782)
- pmid: PMID:31046412
- def: An injury consisting of a laterally displaced scapula with separation of the ipsilateral acromioclavicular joint, disruption of the ipsilateral subclavian vessels and brachial plexus, and an intact overlying skin.
- comment: Scapulothoracic dissociation may be associated with a laterally displaced scapula with an ipsilateral distracted clavicular fracture (most common type), acromioclavicular joint separation or less commonly sternoclavicular joint disruption, without or with concomitant injuries to the ipsilateral subclavian or axillary vessels and the brachial plexus.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11076
- synonyms:
